src/Model/Endpoint.php

```diff
@@ -136,7 +150,7 @@ class Endpoint implements RepositoryInterface, EventListenerInterface, EventDisp
      *
      * @param array $config List of options for this endpoint
      */
-    public function __construct(array $config = [])
+    final public function __construct(array $config = [])
```

Comment:
```suggestion
    public function __construct(array $config = [])
```

Reply:
Reverting this change breaks EndpointLocator at line 96, where we then cannot be sure that we actually instantiate an endpoint. Do you have a suggestion to solve this in another way?

Reply:
I don't see how it breaks the `EndpointLocator`. The core's `Table` class isn't `final` and there are new issues in it's `TableLocator`.

Reply:
Interesting, I'll look into how it's done in core, and update the code to work like that instead.

Reply:
Solved this by adding a psalm constraint, as dictated by psalm. This solution thus only works if any project dependent on Muffin also uses psalm.